paxexpress_cli/versions/models.py

```diff
@@ -58,6 +58,18 @@ class VersionGetResponseModel(VersionModel):
     pass
 
 
+class VersionSummaryModel(BaseModel):
+    name: str
+    repo: Optional[str]
+    package: Optional[str]
+    owner: str
+    released: Optional[datetime.datetime]
+
+
+class VersionGetAvailableVersionResponseModel(BaseModel):
+    __root__: List[VersionSummaryModel] = []
```

Comment:
If this is a `ResponseModel`, why does it not include any data that is usually seen with HTTP responses, e.g `status` or `msg` or `error`

Why doesn't it extend  `GenericResponseModel`

Maybe add a comment to clarify?